spec/lib/services/nodes-api-service-spec.js

```diff
@@ -0,0 +1,251 @@
+// Copyright 2015, EMC, Inc.
+/* jshint node:true */
+
+"use strict";
+
+describe("Http.Services.Api.Nodes", function () {
+    var nodeApiService;
+    var Errors;
+    var taskGraphProtocol;
+    var waterline;
+
+    var computeNode = {
+        id: '1234abcd1234abcd1234abcd',
+        name: 'computeNode',
+        relations: [
+            {
+                "relationType": "enclosedBy",
+                "targets": [
+                    "1234abcd1234abcd1234abcf"
+                ]
+            }
+        ]
+    };
+    var enclosureNode = {
+        id: '1234abcd1234abcd1234abcf',
+        name: 'enclosureNode',
+        relations: [
+            {
+                "relationType": "encloses",
+                "targets": [
+                    "1234abcd1234abcd1234abcd",
+                    "1234abcd1234abcd1234abce"
+                ]
+            }
+        ]
+    };
+    var enclosureNode1 = {
+        id: '1234abcd1234abcd1234abcf',
+        name: 'enclosureNode',
+        relations: [
+            {
+                "relationType": "encloses",
+                "targets": [
+                    "1234abcd1234abcd1234abcd"
+                ]
+            }
+        ]
+    };
+
+    before("Http.Services.Api.Nodes before", function() {
+        helper.setupInjector([
+            helper.require("/lib/services/nodes-api-service")
+        ]);
+        nodeApiService = helper.injector.get("Http.Services.Api.Nodes");
+        Errors = helper.injector.get("Errors");
+        taskGraphProtocol = helper.injector.get("Protocol.TaskGraphRunner");
+        waterline = helper.injector.get('Services.Waterline');
+        waterline.nodes = {
+            needByIdentifier: function() {},
+            updateByIdentifier: function() {},
+            destroy: function() {}
+        };
+        waterline.catalogs = {
+            destroy: function() {}
+        };
+        waterline.workitems = {
+            destroy: function() {}
+        };
+        waterline.lookups = {
+            update: function() {}
+        };
+    });
+
+    beforeEach("Http.Services.Api.Nodes beforeEach", function() {
+        this.sandbox = sinon.sandbox.create();
```

Comment:
This was an error that I put in a bunch of tests a while back. This statement can actually go in the before block instead of the beforeEach block, since we only need to create the sandbox once.